src/utils/protomessage.cc

```diff
@@ -18,184 +18,67 @@ uint64_t ReadVarInt(const char** iter, const char* const end) {
   throw Exception("The file seems truncated.");
 }
 
-std::string EncodeVarInt(std::uint64_t val) {
-  std::string res;
-  while (true) {
-    char c = (val & 0x7f);
-    val >>= 7;
-    if (val) c |= 0x80;
-    res += c;
-    if (!val) return res;
+// // Kept for serialization part.
```

Comment:
I'm unclear what this means, is it that you are intending to write a serialization part in a future PR?